package.json

```diff
@@ -15,11 +18,16 @@
   "author": "jembi",
   "license": "MPL-2.0",
   "devDependencies": {
-    "eslint": "^1.5.1",
-    "jest": "^27.5.1"
+    "@babel/core": "^7.20.2",
+    "@babel/eslint-parser": "^7.19.1",
+    "@types/chai": "^4.3.4",
+    "@types/mocha": "^10.0.0",
+    "eslint": "^8.27.0"
   },
   "dependencies": {
-    "request": "^2.63.0"
+    "chai": "^4.3.7",
+    "mocha": "^10.1.0",
```

Comment:
These are devDependencies

Reply:
Done in [62db86e](https://github.com/jembi/openhim-mediator-utils/pull/10/commits/62db86e286c14761bcc839f1819a366834b09fa5)